app/mailers/mailing_list_entry_notifier_mailer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class MailingListEntryNotifierMailer < ApplicationMailer
+  TO = 'joshua.dedevadas@gmail.com'
+  CC = %w[me@luk4s.dev philipp@thefehr.me].freeze
+  FROM = "MyZivi Mailing-List Notifier <#{ENV['MAIL_SENDER']}>"
```

Comment:
Shouldn't this be a env? I mean now our email addresses are public on github?

Reply:
Well, they're anyway public through commits 😂

Reply:
Yeah, but...

Reply:
Yes?